nilearn/decoding/decoder.py

```diff
@@ -330,6 +330,18 @@ def __init__(self, estimator='svc', mask=None, cv=10, param_grid=None,
         self.n_jobs = n_jobs
         self.verbose = verbose
 
+    def _check_estimator(self):
+        if not isinstance(self.estimator, _basestring):
```

Comment:
We are not aiming for Python2 compatibility, `_basestring` is unnecessary. Use the standard `str`.
I know you didn't write this code, but now we can improve it, here and elsewhere..